modules/rules_spring/2.4.2/presubmit.yml

```diff
@@ -0,0 +1,36 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  - windows
+  bazel: [6.x, 7.x]
```

Comment:
Change to `bazel: [7.x]` or `bazel: [7.x, rolling]`

Reply:
Also dropped _windows_ from the matrix, because our current implementation is not officially supported on Windows.